src/config.ts

```diff
@@ -32,15 +32,42 @@ export interface Config {
 const DEFAULTS: Config = {
   output: 'dist',
   pluginsDirectory: path.join(os.homedir(), '.cdk8s', 'plugins'),
+  imports: ['k8s'],
 };
 
 export function readConfigSync(): Config {
   let config: Config = DEFAULTS;
   if (fs.existsSync(CONFIG_FILE)) {
+    const parsedYaml: Config = yaml.parse(fs.readFileSync(CONFIG_FILE, 'utf-8'));
     config = {
       ...config,
-      ...yaml.parse(fs.readFileSync(CONFIG_FILE, 'utf-8')),
+      ...parsedYaml,
     };
   }
   return config;
+}
+
+export async function addImportToConfig(source: string) {
+  let curConfig = readConfigSync();
+
+  if (!curConfig.imports?.includes(source)) {
+
+    const importsList = curConfig.imports ?? [];
+    importsList.push(source);
+    let config = {
+      ...curConfig,
+      imports: importsList,
+    };
+
+    // just adds the new import if the cdk8s.yaml file already exists
+    // if the cdk8s.yaml does not exist, then it creates it from the defaults
+    if (fs.existsSync(CONFIG_FILE)) {
```

Comment:
Not sure I understand why is this needed - its already covered in `readConfigSync` - no?